resources/shaders/ambient_occlusion.hlsl

```diff
@@ -79,7 +79,7 @@ void AORaygenEntry()
         ao_value -= (1.0f/float(sample_count)) * TraceAORay(0, wpos, getCosHemisphereSample(rand_seed, normal), radius, 0);
     }
 
-    output[DispatchRaysIndex().xy].x = ao_value * power;
+    output[DispatchRaysIndex().xy].x = ao_value / power;
```

Comment:
Higher powers used to decrease the strength of the occlusion since lower values meaning less light thus higher occlusion